test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -2316,6 +2316,703 @@ public async Task GetInstalledPackages_WithNominationUpdate_ReloadsCache()
             }
         }
 
+        [Fact]
+        public async Task TestPackageManager_PreviewProjectsUninstallPackageAsync_AllProjects_Success()
```

Comment:
Consider the What_UnderWhichConditions_ExpectedREsult pattern. 

Now you are slightly missing the under which conditions part. +

Reply:
Ok. Addressed.